### PR TITLE
Add Loukotka-1954-45

### PR DESCRIPTION
**Pull request by Kristina-Pianykh**
# Pull request checklist

- [x] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information
See the discussion in #891. Although this list contains just 45 items, there are two potentially problematic mappings:

1. Loukotka-1954-45-24
2. Loukotka-1954-45-34

The English and Spanish glosses diverge rather drastically in meaning in both cases. Initially the list was compiled in Spanish and only afterwards translated into English.

<details>
<summary>Click to expand</summary>

```
concepticondata\conceptlists\Loukotka-1954-45.tsv
Check: unique_concepticon_gloss
CONCEPTICON_ID      LINE_NO  ID                     NUMBER  GLOSS
----------------  ---------  -------------------  --------  ---------------
                         18  Loukotka-1954-45-17        17  bow, or blowgun
                         25  Loukotka-1954-45-24        24  wildcat
Check: unique_id
OK
Check: unique_number
OK
Check: matching_concepticon_gloss_and_id
OK
Check: valid_concepticon_gloss
OK
```

</details>

### CODE REVIEW

**Comment by Kristina-Pianykh** [comment]
@LinguList could you have a look at it? I was wondering whether to put 1954 (i.e. the year of Rowe's publication) or 1944 (the initial publication of Loukotka) in the column "YEAR" in `concepticonlists.tsv`. I opted for 1954, following the example of Yakhontov's lists.

**Comment by LinguList** [comment]
Yes, @Kristina-Pianykh, I'd do the same. I can edit this, but won't have time until Monday, yet I think I should really do the review, so I'll schedule it and then get back to everybody, or quickly review myself, as the list is short. Thanks a lot already!

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -333,3 +333,4 @@ Pepper-2020-100	Pepper, Steve	2020		100	specific	English	Global		Pepper2020		The
...
+Loukotka-1954-45	Rowe, John Howland	1954		45	basic, areal	Spanish, English	South American languages		Rowe1954	Rowe1954	The given 45-item list was used in a comparative study of South American languages and served as the basis for their preliminary classification ([Loukotka 1944](:bib:Loukotka1944)). Since the list is not readily available as it was not officially provided by the author, it was retrieved from [Rowe 1954](:bib:Rowe1954). Moreover, the study contains a lot of unpublished material, and Loukotka gives no discussion of the evidence for the classification which is claimed to be based on a rather rudimentary method.	24
```
Comment (LinguList) [inline concepticondata/conceptlists.tsv:336]: "The given" -> "This"

File: concepticondata/conceptlists/Loukotka-1954-45.tsv [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Loukotka-1954-45-15	15	house	casa	1252	HOUSE
+Loukotka-1954-45-16	16	pot	olla	1462	POT
+Loukotka-1954-45-17	17	bow, or blowgun	areo, o bodoquera		
+Loukotka-1954-45-18	18	arrow	flecha	977	ARROW
+Loukotka-1954-45-19	19	ax	hacha	677	AXE
+Loukotka-1954-45-20	20	canoe	canoa	1970	CANOE
+Loukotka-1954-45-21	21	man	hombre	1554	MAN
+Loukotka-1954-45-22	22	woman	mujer	962	WOMAN
+Loukotka-1954-45-23	23	dog	perro	2009	DOG
+Loukotka-1954-45-24	24	wildcat	tigre
```
Comment (LinguList) [inline concepticondata/conceptlists/Loukotka-1954-45.tsv:25]: Hehe, this is a classic. You remember Yaguar in Huber et al? I think I'd really be bold here and link to Yaguar, for consistency, as this is what was probably later then used by other lists. It is about a big cat thing in the woods.

Comment (LinguList) [reply concepticondata/conceptlists/Loukotka-1954-45.tsv:25]: But one may add a note here, and also mention Huber, where the same thing occurs.

File: concepticondata/conceptlists/Loukotka-1954-45.tsv [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Loukotka-1954-45-8	8	water	agua	948	WATER
+Loukotka-1954-45-9	9	fire	fuego	221	FIRE
+Loukotka-1954-45-10	10	sun	sol	1343	SUN
+Loukotka-1954-45-11	11	moon	luna	1313	MOON
+Loukotka-1954-45-12	12	star	estrella	1430	STAR
+Loukotka-1954-45-13	13	earth	tierra	1228	EARTH (SOIL)
+Loukotka-1954-45-14	14	stone	piedra	857	STONE
+Loukotka-1954-45-15	15	house	casa	1252	HOUSE
+Loukotka-1954-45-16	16	pot	olla	1462	POT
+Loukotka-1954-45-17	17	bow, or blowgun	areo, o bodoquera
```
Comment (LinguList) [inline concepticondata/conceptlists/Loukotka-1954-45.tsv:18]: let's simplify it and just link to "bow" here. The list is not that important in terms of data, so no real harm.

File: concepticondata/references/references.bib [hunk 4]
```diff
@@ -3390,3 +3390,28 @@ @Phdthesis{Pepper2020
   Year                                = {2020},
   Address                             = {Oslo}
 }
+
+@Article{Loukotka1944,
```
Comment (LinguList) [inline concepticondata/references/references.bib:3394]: Did we SEE this article? I haven't, and if not, this needs to be mentioned in the NOTE. That we did not see the original.

Comment (Kristina-Pianykh) [reply concepticondata/references/references.bib:3394]: I found the actual article. I'll send you the pdf.

File: concepticondata/conceptlists/Loukotka-1954-45.tsv (hunk 2 above)
Comment (Kristina-Pianykh) [reply concepticondata/conceptlists/Loukotka-1954-45.tsv:25]: That's what I thought too. I'll leave a note then in the description.

**Comment by chrzyki** [comment]
Sure, I'll do this together with the PDF for Miller.

**Comment by Kristina-Pianykh** [comment]
I have already made a note about the upload in `conceptlists.tsv` so I will simply merge this PR then.

**Pull request merged**